### ISSUE TITLE
I am getting 404 error while running

### ISSUE THREAD

**Issue by PriyaJayabalan** [arpitaggarwal/spring-social#1]
Hi,

I just downloaded it and  run the same in STS , i am getting 404 error.. is anything to be modified? except of the application.properties.

i have changed the jdk version to 1.8 in project facets since i am using it that is the only thing i modified

**Comment by arpitaggarwal** [comment]
Does application properly deployed in your container (for example - tomcat ) and also what is the url you are trying to access the application?

**Comment by PriyaJayabalan** [comment]
Yes it deployed properly in tomcat and the url is http://localhost:8220/SpringSocial

**Comment by arpitaggarwal** [comment]
Can you try with url `http://localhost:8220/SpringSocial/index.jsp`?

**Comment by PriyaJayabalan** [comment]
I tried it too... It  doesn't work
